src/docs/markdown/caddyfile/directives/tracing.md

```diff
@@ -0,0 +1,53 @@
+---
+title: tracing (Caddyfile directive)
+---
+
+# tracing
+
+It provides integration with OpenTelemetry tracing facilities.
+
+When enabled, it will propagate an existing trace context or initialize a new one.
+
+It is based on `https://github.com/open-telemetry/opentelemetry-go`.
```

Comment:
Maybe make this a link. (And is that even relevant for the users of the server?)

Reply:
Good idea 👍. I made it a link. 

There are a few reasons why I would like to keep it here. 

Most important to me is that `open-telemetry/opentelemetry-go` at the moment does not support some of the environment variables that are used for a configuration (https://github.com/open-telemetry/opentelemetry-go/issues/1698). 

Having this reference here gives some extra information for the webserver users why one or another feature doesn't work.